### ISSUE TITLE
feature request: accept pipe of json lines and tolerance of non json

### ISSUE THREAD

**Issue by kokokenada** [PaulJuliusMartinez/jless#102]
Context:
* whenever possible, our project outputs JSON for console.log (error, warn, debug, info) in JSON (json lines).  The primary motivation was better log searching and display on the cloud infrastructure. 
* Sometimes dependencies output non JSON
* In our local development environment, we tail the output and pretty print the JSON

What would be cool:
* If one could `npm run start:watch | jless --console-debug`
* When a JSON line is output, developers could interactively drill and explore the JSON
* When a non JSON line is given, it would just pass it along to stdout
* When `jess -l  1` was given, the JSON would be expanded by default to the first level only (sometimes when debugging, one outputs a huge json object which is hard to scroll through)

This would make server debugging similar to debugging in the browser console, where you can console.log(someObject) and can then expand and explore the object.

**Comment by neilotoole** [comment]
Agreed on the need for this. My use case is similar... I have an OSS project [sq](https://sq.io) that outputs JSON logs. I want to be able to tail/follow those logs. Something like:

```shell
$ tail -f sq.log | jless

# or even
$ jless -f sq.log
```

**Comment by aexvir** [comment]
joining here also to request this, in my use case our service outputs json logs, then I pipe that to `jq` to only select the ones I care about, and I need some nice way to inspect the output of that in a continuous stream, so I'm doing something like
```bash
service | jq --unbuffered '. | select(.event | endswith("debug.err")) | .["error.message"]' | jless
```
but sadly it doesn't work

**Comment by googleson78** [comment]
+1 to this
